### ISSUE TITLE
Report file path when there's an xml loading error

### ISSUE THREAD

**Issue by nguerrera** [dotnet/xliff-tasks#49]
Invalid xml source files should not be reported as an unexpected task failures and error messages should include the path to the problematic file:

Example of unfortunate error reporting today for invalid input xml:
```
C:\...\XliffTasks.targets(90,5): error MSB4018: The "UpdateXlf" task failed unexpectedly. [C:\...\BasicCodeAnalysis.vbproj]
C:\...\XliffTasks.targets(90,5): error MSB4018: System.Xml.XmlException: The 'data' start tag on line 5527 position 4 does not match the end tag of 'root'. Line 5544, position 3. [C:\...\BasicCodeAnalysis.vbproj]
<stack trace>
```

**Comment by nguerrera** [comment]
This is a subset of #13 that's easier to implement. Maybe start by fixing this.

**Comment by tmeschter** [comment]
Fixed by #57.
